### PR TITLE
Alternate lab order print/display layout

### PR DESCRIPTION
**Pull request by mdsupport**

### CODE REVIEW

File: interface/orders/single_order_results.inc.php [hunk 1]
```diff
@@ -358,7 +385,8 @@ function showpnotes(orderid) {
  for (; w.name != 'RTop' && w.name != 'RBot'; w = w.parent) {
   if (w.parent == w) {
    // This message is not translated because a developer will need to find it.
-   alert('Internal error locating target frame in ' + (window.opener ? 'opener' : 'window'));
+   top.RTop.location = "<?php echo $GLOBALS['webroot']; ?>/interface/patient_file/summary/pnotes_full.php?orderid=" + orderid;
```
Comment (mdsupport) [inline interface/orders/single_order_results.inc.php]: Temporary fix for issue #383

**Comment by bradymiller** [comment]
It looks like there is less information on this output (which would likely not go well with MU). Is there a way to make this new layout an option via a global?

**Comment by mdsupport** [comment]
Can you identify what is missing?  In lab results there is not much to eliminate.

**Comment by bradymiller** [comment]
Sounds like I need to look at this in more details. I saw the following removed on my quick review:
``````
<td><?php echo xlt('Reported'); ?></td>
 -  <td><?php echo xlt('Collected'); ?></td>
 -  <td><?php echo xlt('Specimen'); ?></td>
 -  <td><?php echo xlt('Status'); ?></td>
 -  <td><?php echo xlt('Note'); ?></td>
 -  <td><?php echo xlt('Name'); ?></td>

**Comment by bradymiller** [comment]
Also looks like pnotes have been removed which were needed to fulfill MU2 (as I recall, used for linking and reports of imaging).

**Comment by mdsupport** [comment]
All of these are present - just getting reformatted or moved around so openemr report looks and feels like that send by most high volume labs.  That is a huge issue when sending patient records to other practices.  

Here is LabCorp:
image

Here is Quest:
image

**Comment by mdsupport** [comment]
Based on developers' input, we are going to withdraw this because of internal framework switch.  That will complicate future maintenance of old code.  Sorry.

**Comment by sunsetsystems** [comment]
I'm not understanding the reason for closing this.  Also do you have a before-and-after screenshot?

**Comment by mdsupport** [comment]
### Current layout:
image
..
image


### PR layout:

image
..
image

**Comment by sunsetsystems** [comment]
Offhand this looks reasonable.  Are you still open to a review?  If so I'd like to get Stephen Waite to weigh in from the user perspective.

**Comment by mdsupport** [comment]
We are in the process of switching the source of these reports from db tables to hl7 structured data.  So major changes from review may not be done in a timely manner.

**Pull request closed without merging**